### PR TITLE
refactor to use partial initial values

### PR DESCRIPTION
**Pull request by sunghwan-park-fw**

### CODE REVIEW

**Comment by sunghwan-park-fw** [comment]
@vyasworks 
I don't understand this pr-check failure. This branch is based on 'happ-441_severity-of-cough' branch and it has passed pr-check.

**Comment by vyasworks** [comment]
> @vyasworks I don't understand this pr-check failure. This branch is based on 'happ-441_severity-of-cough' branch and it has passed pr-check.

you can open the action and see it says 

```src/forms/SymptomChecker.tsx
  Line 6:10:  'Values' is defined but never used  @typescript-eslint/no-unused-vars```

File: apps/web/src/types/Symptoms.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Severity } from './Severity';
+
+export class Symptoms {
+  fever!: Severity;
```
Comment (hitankar-freshworks) [inline apps/web/src/types/Symptoms.ts:4]: why not? by default it would be false

```suggestion
  fever!: Severity | boolean = false;
```

**Review by hitankar-freshworks — CHANGES_REQUESTED** [review]
Please take a look at my comment

**Review by hitankar-freshworks — APPROVED** [review]

**Pull request merged**